functions/helloworld/hello-http

Clean download, `mvn clean package` results:
```
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.013 s <<< FAILURE! - in functions.ExampleSystemTest
[ERROR] functions.ExampleSystemTest.helloHttp_shouldRunWithFunctionsFramework  Time elapsed: 0.002 s  <<< ERROR!
java.lang.IllegalArgumentException: URI with undefined scheme
	at java.net.http/jdk.internal.net.http.common.Utils.newIAE(Utils.java:244)
	at java.net.http/jdk.internal.net.http.HttpRequestBuilderImpl.checkURI(HttpRequestBuilderImpl.java:79)
	at java.net.http/jdk.internal.net.http.HttpRequestBuilderImpl.uri(HttpRequestBuilderImpl.java:71)
	at java.net.http/jdk.internal.net.http.HttpRequestBuilderImpl.uri(HttpRequestBuilderImpl.java:43)
	at functions.ExampleSystemTest.helloHttp_shouldRunWithFunctionsFramework(ExampleSystemTest.java:44)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:364)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:272)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:237)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:158)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)

[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   ExampleSystemTest.helloHttp_shouldRunWithFunctionsFramework:44 » IllegalArgument
[INFO]
[ERROR] Tests run: 5, Failures: 0, Errors: 1, Skipped: 0
```

--- a/functions/helloworld/hello-http/src/test/java/functions/ExampleSystemIT.java
+++ b/functions/helloworld/hello-http/src/test/java/functions/ExampleSystemIT.java
@@ -40,6 +40,14 @@ public class ExampleSystemIT {
   public void helloHttp_shouldRunWithFunctionsFramework() throws IOException, InterruptedException {
     String functionUrl = BASE_URL + "/HelloHttp";
 
+    // [END functions_http_system_test]
+    // Skip this test if FUNCTIONS_BASE_URL is not set
+    if (BASE_URL == null) {
+      System.out.println("FUNCTIONS_BASE_URL is not set; skipping ExampleSystemIT.");
+      return;
+    }
+
+    // [START functions_http_system_test]
     java.net.http.HttpRequest getRequest =
         java.net.http.HttpRequest.newBuilder().uri(URI.create(functionUrl)).GET().build();
 

